tasks/inspect.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- name: Obtain the PowerDNS version
+  shell: "pdns_server --version 2>&1 | awk '/PowerDNS Authoritative/{print $7}'"
+  register: pdns_version_output
+
+- debug:
+    msg: "{{ pdns_version_output }}"
+
+- set_fact:
+    pdns_package_version: "{{ pdns_version_output['stdout'] }}"
```

Comment:
does this need an `rstrip()`?

Reply:
Why?

Reply:
stdout might contain a `\n`

Reply:
Yes. But it's not the case of the string we're using to extract the version from with `awk`. Column `$7` is not the last one :)